membership-attribute-service/app/models/Attributes.scala

```diff
@@ -83,6 +89,7 @@ case class DynamoAttributes(
   MembershipJoinDate: Option[LocalDate] = None,
   DigitalSubscriptionExpiryDate: Option[LocalDate] = None,
   PaperSubscriptionExpiryDate: Option[LocalDate] = None,
+  GuardianWeeklySubscriptionExpiryDate: Option[LocalDate] = None,
```

Comment:
we never actually write this field into dynamo! so cached responses to guardian weekly subscribers cause support asks to appear
https://github.com/guardian/members-data-api/blob/9970f2e01cd60d21abb0b77c7207b70785882746/membership-attribute-service/app/services/ScanamoAttributeService.scala#L58-L62

Reply:
wow good find. is someone going to raise a PR to address this?

Reply:
There was actually a determination from the business a year-or-so ago that Guardian Weekly subscription would not count towards a support ask. It's worth reconfirming that before we wire up to the boolean which takes down asks. However, `guardianWeeklySubscriber` Content Access should still be stored in the Dynamo cache.

Reply:
yes we are going to fix it.. but it shows an issue with the structure of the code that there's no hint that the write and read paths are in completely different places!  And no tests that picked that up.
I thought I would document it here in case someone is referring to this in future!  But I will at least add a comment in the code if nothing stronger.

Reply:
@paulbrown1982 It's already wired up, it's been like that the whole time and it would work if the attributes don't come from the cache.
https://github.com/guardian/members-data-api/blob/26e95187feeb2431c705f75825d822453d2f4811/membership-attribute-service/app/models/Attributes.scala#L72
if it was decided not to include it at some point more recently, no one actually implemented it by the looks.
If someone confirms that then it's very easy to change the above line.  But we also need to let userhelp know.

Reply:
OK if it's already in there it must be OK by now! Let's just get Dynamo working properly.

Reply:
https://github.com/guardian/members-data-api/pull/482